### DISCUSSION TITLE
More article info in List View

### DISCUSSION THREAD

**Discussion by tbeseda** [dailydotdev/daily#404] [Feature Requests]
When viewing a list of articles, I'd like to see more info about each item. Is this already an option?
Notably: the article domain and its tags.

A simple mockup:
![daily dev](https://user-images.githubusercontent.com/15697/147689932-39b8218b-3cc8-4cef-947f-fbfe7370a6aa.png)

The site logo doesn't tell me a whole lot unless it's very recognizable or I hover over the logo.

Thanks!

**Comment by idoshamun** [comment]
Hi @tbeseda,

Great suggestion. We're working on providing customizations to the list and grid items.
It might take some time though, stay tuned.
